react/ProductSummarySKUSelector.tsx

```diff
@@ -190,7 +190,7 @@ function ProductSummarySKUSelector(props: Props) {
       ...selectedItem,
       image: selectedItem.images[0],
       seller:
-        getFirstAvailableSeller(selectedItem.sellers) ??
+        getDefaultSeller(selectedItem.sellers as ProductTypes.Seller[]) ??
```

Comment:
I don't know about these types 😆 , but you can avoid doing this cast, doing something like this on `getDefaultSeller`.

```ts
interface MinimumSellerNeededProperties { // This name is ilustrative
  sellerDefault: <sellerDefaultTypeHere> 
}

export function getDefaultSeller<T extends MinimumSellerNeededProperties>(sellers?: T[]): T {}
```

Reply:
Every subtype of MinimumSellerNeededProperties can be used as a parameter and the result of this function also will be typed with the subtype 😃 .

Reply:
We can remove the `?? selectedItem.sellers[0]` the function already does this